osx-utils/performous-app-build.sh

```diff
@@ -30,6 +39,39 @@ function asksure {
 	return $retval
 }
 
+function detectPkgManager {
+	brew_retcode=$(brew -v 1>/dev/null 2>&1; echo $?)
+	port_retcode=$(port version 1>/dev/null 2>&1; echo $?)
+	if [[ ${brew_retcode} = 0 ]]
+		then
+			brew_prefix="$(dirname $(dirname $(which brew)))"
+			echo "Homebrew install found at: ${brew_prefix}"
+			PREFIXDIR="${brew_prefix}"
+	fi
+	if [[ ${port_retcode} = 0 ]]
+		then
+			port_prefix="$(dirname $(dirname $(which port)))"
+			echo "MacPorts install found at: ${port_prefix}"
+			PREFIXDIR="${port_prefix}"
+	fi
+	if [[ ${port_retcode} = 0 && ${brew_retcode} = 0 ]]
+		then
+			if askPrompt "Would you like to use Homebrew (y), or MacPorts (n)?"
+				then
+					PREFIXDIR="${brew_prefix}"
+				else
+					PREFIXDIR="${port_prefix}"
+			fi
+	fi
+}
+
+test -z ${PREFIXDIR} && detectPkgManager # Look for both Homebrew and Macports and use whichever is found. If for some reason, both are available, let the user choose. Alternatively, define this while invoking the script to bypass this behavior.
+test -z ${DEPLOYMENT_TARGET} && DEPLOYMENT_TARGET="10.14" # Change this if you want to target a different version of macOS.
```

Comment:
instead of a static string i think it'd be better to use `sw_vers` for this to make it compatible with all systems instead of changing stuff in the directory. It also prevents useless commits/changes.

Reply:
Fixed. I also squashed everything because the commit list was getting too bloated.